pkg/cli/config.go

```diff
@@ -399,6 +405,10 @@ func (c *Config) token() (string, error) {
 	if c.oauthToken != "" {
 		return c.oauthToken, nil
 	}
+    if c.TokenValue != "" {
```

Comment:
*nit*: indentation seems to be incorrect here

Reply:
Fixed